File: 백준/Bronze/30033. Rust Study/Rust Study.cs
int N = int.Parse(Console.ReadLine());
int cnt = 0;

string[] A = Console.ReadLine().Split();
string[] B = Console.ReadLine().Split();

for (int i = 0; i < N; i++)
{
    int a = int.Parse(A[i]);
    int b = int.Parse(B[i]);

    if (b >= a)
    {
        cnt++;
    }
}

Console.WriteLine(cnt);

File: 백준/Bronze/30033. Rust Study/Rust Study.py
N = int(input())
cnt = 0

A = list(map(int, input().split()))
B = list(map(int, input().split()))

for i in range(N):
    a = A[i]
    b = B[i]

    if b >= a:
        cnt += 1

print(cnt)
